R/cards-package.R

```diff
@@ -1,6 +1,6 @@
 #' @keywords internal
 #' @import rlang
-#' @importFrom dplyr across
+#' @importFrom dplyr across cur_column rows_update if_any if_all
```

Comment:
- `cur_column()` is being imported, but only used once and it has its `dplyr::` prefix, so I don't think we need to import it.
- Same for `rows_update()`
- Same for `if_any()`
- Same for `if_all()`

Reply:
good point...fixed!